packages/block-library/src/query/editor.scss

```diff
@@ -43,3 +43,12 @@
 		max-height: none;
 	}
 }
+
+.block-library-query-toolspanel__filters {
+	.components-form-token-field__help {
+		margin-bottom: 0;
+	}
+	.block-library-query-inspector__taxonomy-control:not(:last-child) {
+		margin-bottom: calc(4px * 6);
```

Comment:
This value matches the default ToolsPanel spacing([here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/ui/utils/space.ts#L46) and [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/tools-panel/styles.ts#L22)). Is there a better way to do this?

Reply:
We'll probably make some kind of component abstraction to handle this standard Inspector spacing easily (related: #39358), but for now I think you can use `$grid-unit-30` here. (Just for easier discovery once we are ready to start refactoring everything.)

Reply:
Using `$grid-unit-30` here sounds like a good idea to me as well.

Reply:
Thanks!